test/functional/test_framework/blocktools.py

```diff
@@ -217,3 +219,11 @@ def send_to_witness(use_p2wsh, node, utxo, pubkey, encode_p2sh, amount, sign=Tru
             tx_to_witness = ToHex(tx)
 
     return node.sendrawtransaction(tx_to_witness)
+
+class TestFrameworkScript(unittest.TestCase):
+    def test_createcoinbase(self):
+        next_height = 20
+        coinbase_tx = create_coinbase(height=next_height)
+        # round-trip the encoded bip34 block height commitment
+        assert_equal(CScriptNum.decode(coinbase_tx.vin[0].scriptSig), next_height)
+        
```

Comment:
Added this newline, will update with next batch of changes.

Reply:
Btw, if you install `flake8`, you can run the python linter with `test/lint/lint-python.sh`. It has saved me a lot of force pushes 😂